modules/apps/object/object-test/src/testFunctional/tests/ObjectRootModel.testcase

```diff
@@ -21,6 +21,76 @@ definition {
 		}
 	}
 
+	@description = "LPS-189380 - Verify the user can to see relationships tabs in the initial out-of-the-box layout of one level of children"
```

Comment:
You're duplicating the description here. It should be: _"Description LPS-189380 - Verify published child object definitions do not have dedicated portlets and the related configurations are read only"_  as stated in the ticket

Reply:
Done [here](https://github.com/liferay-bpm-qa/liferay-portal/pull/1202/files#diff-03f0ca92d34d22561a353c49a77883f8a25588876d02cc52f3e4686e20d43188R24)